Reference/conformance/conformance.pb.swift

```diff
@@ -230,11 +230,28 @@ struct Conformance_ConformanceResponse: SwiftProtobuf.Message {
   var unknownFields = SwiftProtobuf.UnknownStorage()
 
   enum OneOf_Result: Equatable {
```

Comment:
The names of the cases are the same as the names of the fields, right? So I don't think it's necessary, especially since the oneof enum is always tightly coupled to the message it's in.